src/e2e-test/features/bigqueryexecute/BQExecute.feature

```diff
@@ -0,0 +1,175 @@
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Enter textarea plugin property: "sql" with value: "bqExecuteDDLCreate"
+    Then Override Service account details if set in environment variables
+    Then Validate "BigQuery Execute" plugin properties
+    Then Close the Plugin Properties page
+    Then Save and Deploy Pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Verify BigQuery table: "bqExecuteCreateTable" is created
+
+  @BQ_SOURCE_BQ_EXECUTE_TEST @BQ_EXECUTE_DDL_DROP_TEST
+  Scenario: Verify BQExecute plugin functionality for DDL query - Drop table
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "Big Query Execute" from the plugins list as: "Conditions and Actions"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Enter textarea plugin property: "sql" with value: "bqExecuteDDLDrop"
+    Then Override Service account details if set in environment variables
+    Then Validate "BigQuery Execute" plugin properties
+    Then Close the Plugin Properties page
+    Then Save and Deploy Pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Verify BigQuery table: "bqSourceTable" is deleted
+
+  @BQ_SOURCE_BQ_EXECUTE_TEST @BQ_EXECUTE_INSERT_SQL
+  Scenario: Verify BQExecute plugin functionality for DML query - Insert data
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "Big Query Execute" from the plugins list as: "Conditions and Actions"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Enter textarea plugin property: "sql" with value: "bqExecuteDMLInsert"
+    Then Override Service account details if set in environment variables
+    Then Validate "BigQuery Execute" plugin properties
+    Then Close the Plugin Properties page
+    Then Save and Deploy Pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Verify 1 records inserted in BigQuery table: "bqSourceTable" with query "bqExecuteCountDMLInsert"
+
+  @BQ_SOURCE_TEST @BQ_EXECUTE_UPSERT_SQL
+  Scenario: Verify BQExecute plugin functionality for DML query - Upsert data
```

Comment:
Can use a pipeline similar to the screenshot to simulate upsert, one plugin inserts and other plugin update/insert records, this will not need to create 2 pipelines?? @rmstar wdyt??

![image](https://user-images.githubusercontent.com/88528384/194865576-1fa0b09d-c4a6-4102-bf9f-ca9b2e032601.png)

Reply:
We will not be able to validate first plugins result with above pipeline.

We should run pipeline twice to validate UPSERT query functionality.
In first run record should be inserted. In Second run same record should be updated.